BREAKING.md

```diff
@@ -18,11 +18,17 @@ There are a few steps you can take to write a good change note and avoid needing
 
 ## 0.60 Upcoming changes
 - [Summarize heuristic changes based on telemetry](#Summarize-heuristic-changes-based-on-telemetry)
+- [bindToContext to be removed from IFluidDataStoreChannel](#bindToContext-to-be-removed-from-IFluidDataStoreChannel)
 
 ### Summarize heuristic changes based on telemetry
 Changes will be made in the way heuristic summaries are run based on observed telemetry (see `ISummaryConfigurationHeuristics`). Please evaluate if such policies make sense for you, and if not, clone the previous defaults and pass it to the `ContainerRuntime` object to shield yourself from these changes:
 - Change `minOpsForLastSummaryAttempt` from `50` -> `10`
 
+### bindToContext to be removed from IFluidDataStoreChannel
+`bindToContext` will be removed from `IFluidDataStoreChannel` in the next major release.
+It was deprecated in 0.50 but due to [this bug](https://github.com/microsoft/FluidFramework/issues/9127) it still had to be called after creating a non-root data store. The bug was fixed in 0.59.
+These calls to `bindToContext` should now be removed in this version since N / N-1 compatibility across container runtime and data store runtime boundary will be maintained. Basically, container runtime version 0.60 running with data store runtime version 0.59 will not this bug.
```

Comment:
```suggestion
To prepare for the removal in the following release, calls to `bindToContext` can and should be removed as soon as this version is consumed, since the compatibility window between container runtime and data store runtime is N / N-1 meaning all runtime code will have the required bug fix (released in the previous version 0.59).
```